src/Defra.PTS.Sql/PostDeployment/Script.PostDeployment.BreedUpdate.sql

```diff
@@ -12,16 +12,26 @@ Post-Deployment Script Template
 GO
 SET IDENTITY_INSERT [dbo].[Breed] ON 
 GO
-IF NOT EXISTS (SELECT * FROM [dbo].[Breed] Where Id = 99 AND [Name] = 'Mixed breed or unknown')
+IF EXISTS (SELECT * FROM [dbo].[Breed] Where Id = 99 AND [Name] = 'Mixed breed or unknown')
 BEGIN
-	INSERT [dbo].[Breed] ([Id], [Name], [SpeciesId]) VALUES (99, N'Mixed breed or unknown', 1)
+	UPDATE [dbo].[Breed] SET [Name] = 'Mixed-breed or unknown'
```

Comment:
This will update all the rows in breed table as ther si not where clause. Is that expected behaviour

Reply:
My bad. Added WHERE clause now.